sklearn/cross_decomposition/_pls.py

```diff
@@ -783,6 +785,15 @@ class PLSSVD(TransformerMixin, BaseEstimator):
         potentially scaling. If False, these operations will be done inplace,
         modifying both arrays.
 
+    random_state : int, RandomState instance or None, optional (default=None)
+        The seed of the pseudo random number generator to use when shuffling
+        the data.  If int, random_state is the seed used by the random number
```

Comment:
WE are using the random_state here to initialize the starting vector for `svds`.

Reply:
Done